parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -231,12 +244,11 @@ extractHyperkube() {
 
 installKubeletAndKubectl() {
     if [[ ! -f "/usr/local/bin/kubectl-${KUBERNETES_VERSION}" ]]; then
-        if [[ "$CONTAINER_RUNTIME" == "docker" ]]; then
-            extractHyperkube "docker"
-        else
-            installImg
-            extractHyperkube "img"
-        fi
+      if version_gte ${KUBERNETES_VERSION} 1.17; then  # don't use hyperkube
+          extractKubeBinaries ${KUBERNETES_VERSION}
```

Comment:
The KUBERNETES_VERSION is passed from AKS RP to CSE and to this method, it is only 1.17.3, and cannot be something like 1.17.3-hotfix.20200601.1, so it cannot download the patched ones. I think we should let RP pass in the kubeBinary download URL, similar to HYPERKUBE_URL

Reply:
Check https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp/pullrequest/2807797 you might have a bit more idea